addons/website/static/src/snippets/s_timeline/000.scss

```diff
@@ -1,4 +1,4 @@
-.s_timeline {
+.s_timeline:not([data-vcss]) {
     --spacing: #{$grid-gutter-width * .5};
 
```

Comment:
The alignement issue I was mentioning was this one :arrow_down: 
![image](https://github.com/user-attachments/assets/eb758dd9-1857-4f60-83f3-c87bf181b4fe)

Because of the "Alignment" card option => if the width is not 100%, it keeps the alignment set on desktop view
From what I saw, you just need to define a CSS rule for below the MD screen breakpoint and targeting `s_timeline_card`, setting  `margin-left: 0 !important;` I think (to cancel the margin classes set by the Alignment option) :slightly_smiling_face:

Reply:
I'm not sure this is a good idea 🤔  We should try to avoid `!important` tag as much as possible.
Also, this is the expected behavior: If the user try to change the alignment in mobile, nothing will happen, which is not good.
@stefanorigano what do you think ?

Reply:
@Brieuc-brd The alignment option is the one used to have the card stuck to the "line" or not, when the width is not 100% :wink: So the left card is aligned to the right and the right card is aligned to the left :+1: 
But in mobile, since the timeline is forced on the left of the screen, the cards that were aligned to the left are okay, but the ones that were to the right are shifted to the right, which is why there is this misalignment :slightly_smiling_face: 
And using the alignment option to be well aligned on mobile will change the alignment on desktop so we cannot use it :grimacing: 
Which is why my suggestion is to simply cancel the option behavior (which sets the `me-auto`, `ms-auto` and `mx-auto` classes) for the mobile view, since we only want all the cards to be aligned on the left :slightly_smiling_face: 
For this kind of use case, I think there is nothing wrong to use `!important` since we really want to override the desktop rule (this is already used at other places) :+1:

Reply:
> The alignment option is the one used to have the card stuck to the "line" or not, when the width is not 100% 😉 So the left card is aligned to the right and the right card is aligned to the left 👍 ...

Yes I got the issue, and I understand why you want to do this 🙂 
But I'm still convinced that this is not the right solution.

The problem doesn't come from the card or this snippet, but from the fact that the Web Editor doesn't manage the mobile version perfectly. 
Let's take another example, `s_three_column`:
https://www.awesomescreenshot.com/video/30205967?key=37331a33e8e8dfceafb674373fcb304c

In this video, I want to change the spacing under a card, but in doing so I'll have a spacing issue on mobile that I can't handle with the Web Editor.
However, we haven't forced the padding bottom to 0 to "solve" this problem, because I think we assume that the user may want this spacing for a specific reason.

IMO it's the same thing for `s_timeline`. Although we don't have specific features to manage the mobile version, we shouldn't force the alignment. It's up to the users to manage the design as they want.

As I said previously, even if we decide to force this alignment for cards, this is how it would look like :
https://www.awesomescreenshot.com/video/30206397?key=308567c5e83afab15c4fe705cde3cd17

If the user try to change the alignment in mobile, nothing will happen (which is not good 😬), but even worst he could get an unexpected rendering on desktop since he could change the values in mobile without seeing what he does.

IMO, we should keep it as it is, until the web editor provides a way to manage it on mobile 🙂 .

cc @stefanorigano

Reply:
Okay then, let's keep it like that for now :+1: I will keep that in mind in order to eventually improve the options for mobile views (there are other options too that have issues, as you mentioned with the padding) :slightly_smiling_face: 
And at worst, we can still add the rule later, if some users complain about it :wink: